serdio/serdio/io_lifter.py

```diff
@@ -1,77 +1,25 @@
-"""Tools for lifting normal Python functions into Cape handlers.
-
-We automatically convert between Python functions and Cape handlers by decorating
-a given Python function with a version that deserializes inputs from msgpack-ed bytes,
-executes the original function on those inputs, and then serializes outputs w/ msgpack.
-Custom types are handled by user-supplied encode_hook and decode_hook functions,
-bundled into a SerdeHookBundle dataclass.
-
-Basic usage in app.py: ::
-
-    @serdio.lift_io(as_handler=True)
-    def cape_handler(x: int, y: float) -> float:
-        return x * y
-
-Then with Cape.run: ::
-
-    function_id = "1s25hd1s28f12"
-    cape = Cape()
-    z = cape.run(function_id, 2, 3.0, use_serdio=True)
-    print(z)
-    # 6.0
-
-Usage with custom types: ::
```

Comment:
Where do you plan to have usage for custom types?

Reply:
It's already in the readme